project.clj

```diff
@@ -7,12 +7,21 @@
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0.html"}
 
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.clojure/java.jdbc "0.3.2"]
-                 [org.postgresql/postgresql "9.3-1100-jdbc41"]
-                 [com.jolbox/bonecp "0.8.0.RELEASE" :exclusions [[org.slf4j/slf4j-api]]]
+  :dependencies [[com.jolbox/bonecp "0.8.0.RELEASE" :exclusions [[org.slf4j/slf4j-api]]]
```

Comment:
Whether we ditch bonecp as a part of this is questionable, I'm erring on leaving it - since not everyone might want to bump to the new connection pool, but still want to benefit from other releases of jdbc-util.